stratum/hal/bin/tdi/dpdk/dpdk_main.cc

```diff
@@ -46,7 +47,13 @@ namespace tdi {
 ::util::Status DpdkMain(int argc, char* argv[]) {
   // Default value for DPDK.
   FLAGS_chassis_config_file = CONFIG_PREFIX "dpdk_port_config.pb.txt";
-  InitGoogle(argv[0], &argc, &argv, true);
+  FLAGS_log_dir = LOG_DIR;
```

Comment:
```
// Log to file by default.
```
Do we want to do this in all cases, or only when krnlmon is enabled?

You might want to add an explanation for why we default to this.

Reply:
Irrespective of krnlmon, if user is running infrap4d as background process we need to define where all stratum logging would go.

Reply:
I think the real question is, why does DPDK default to logging to a file when everyone else defaults to logging to the console? And is there a straightforward way for the user to switch back to the console for debugging purposes?